Core/PSPLoaders.cpp

```diff
@@ -355,3 +356,11 @@ bool Load_PSP_ELF_PBP(FileLoader *fileLoader, std::string *error_string) {
 
 	return __KernelLoadExec(finalName.c_str(), 0, error_string);
 }
+
+bool Load_PSP_GE_Dump(FileLoader *fileLoader, std::string *error_string) {
+	BlobFileSystem *umd = new BlobFileSystem(&pspFileSystem, fileLoader, "data.ppdmp");
+	pspFileSystem.Mount("disc0:", umd);
```

Comment:
Well, at this point I was trying to get it working quickly.  Should probably get the filename passed via PSP RAM in `a0` and use a DirectoryFileSystem...

-[Unknown]